copy_to_modules/Styla/metadata.php

```diff
@@ -26,14 +26,14 @@
         'Styla_Magazine'    => 'Styla/controllers/Styla_Magazine.php',
     ),
     'templates'   => array(
-        'Styla_JSON.tpl' => 'Styla/views/tpl/Styla_JSON.tpl',
+        'Styla_JSON.tpl'     => 'Styla/views/tpl/Styla_JSON.tpl',
         'Styla_Magazine.tpl' => 'Styla/views/tpl/Styla_Magazine.tpl',
     ),
     'settings'    => array(
         array('name' => 'styla_username', 'type' => 'str', 'value' => '', 'group' => 'styla_general', 'constraints' => '', 'position' => 1), //
-        array('name' => 'styla_api_url', 'type' => 'str', 'value' => 'http://live.styla.com', 'group' => 'styla_general', 'constraints' => '', 'position' => 3), //
-        array('name' => 'styla_js_url', 'type' => 'str', 'value' => 'http://cdn.styla.com', 'group' => 'styla_general', 'constraints' => '', 'position' => 3), //
-        array('name' => 'styla_seo_server', 'type' => 'str', 'value' => 'http://seo.styla.com', 'group' => 'styla_general', 'constraints' => '', 'position' => 3), //
+        array('name' => 'styla_api_url', 'type' => 'str', 'value' => 'https://live.styla.com', 'group' => 'styla_general', 'constraints' => '', 'position' => 3), //
```

Comment:
should be http since its executed only on server side and might be an issue with ssl depending on the server configuration